build-system/compile/pre-closure-babel.js

```diff
@@ -62,13 +64,12 @@ function preClosureBabel() {
   const babel = gulpBabel({caller: {name: 'pre-closure'}});
 
   return through.obj((file, enc, next) => {
-    const {relative, path} = file;
-    if (!filesToTransform.includes(relative)) {
+    if (!filesToTransform.includes(file.relative)) {
```

Comment:
Too many things with the name `path`. Good idea to not spread here since it's getting confusing which `path` is what and where.

Perhaps in the future an ESLint rule for `build-system` prohibiting the spread of something creating a `path` variable might make sense.

Reply:
Agree. I noticed that some files use `path` to refer to the module, while others use `path` as a local variable and use `pathModule` to refer to the module. Standardizing is out of scope for this PR, but it is worth doing in a future PR.